health-api-library/library/src/main/java/net/gini/android/health/api/response/PaymentProviderResponse.kt

```diff
@@ -12,8 +14,13 @@ internal data class PaymentProviderResponse(
     @Json(name = "colors") val colors: Colors,
     @Json(name = "iconLocation") val iconLocation: String,
     @Json(name = "playStoreUrlAndroid") val playStoreUrl: String?,
-    @Json(name = "gpc_supported") val gpcSupported: Boolean?,
-)
+    @Json(name = "gpcSupportedPlatforms") val gpcSupportedPlatforms: List<String>?,
+    @Json(name = "openWithSupportedPlatforms") val openWithSupportedPlatforms: List<String>?
+) {
+
+    fun isEnabled(): Boolean = (gpcSupportedPlatforms?.map { it.lowercase(Locale.getDefault()) }?.contains(ANDROID_PLATFORM) == true )
```

Comment:
[Request] `Locale.getDefault()` is dangerous here because it can happen that the default locale does not convert to lowercase in the way we expect it. I [found that using](https://stackoverflow.com/a/10336915) `Locale.ENGLISH` is safe for identifiers.

Reply:
Thank you for pointing this out - changed it to use `Locale.ENGLISH`